sda-auth/dev-server/docker-compose.yml

```diff
@@ -69,6 +69,8 @@ services:
     volumes:
       - ../keys:/keys
       - ../:/sda-auth
+      - ../keys/crypt4gh_key.pub.pem:/crypt4gh_key.pub.pem
+      - ../keys/crypt4gh_key.sec.pem:/crypt4gh_key.sec.pem
```

Comment:
```suggestion
```

This is rather pointless since the files are already mounted in the container under the `/keys` path

What's missing though is to set `C4GHPUBPATH=/keys/crypt4gh_key.pub.pem` under `environment:` above.